postgres/16/Dockerfile

```diff
@@ -0,0 +1,25 @@
+# ----------------------------------
+# Environment: Postgres DB
+# ----------------------------------
+FROM --platform=$TARGETOS/$TARGETARCH postgres:16.1-bookworm
```

Comment:
i would use 16-bookworm. this contains Always the latest Version

Reply:
Ok. I habe changed it